### PR TITLE
Re-generate reports at 2021-07-04

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**

Re-generate reports at 2021-07-04

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,16 +85,15 @@ generate-samples: install
 	go run ./hack/samples/generate_samples.go
 
 .PHONY: generate-testdata ## Generate the full testdata directory
-generate-testdata: generate-samples
+generate-testdata:
+	docker login https://registry.redhat.io
+	make generate-samples
```
Comment (bentito) [inline Makefile:90]: Was this not working as a prerequisite? Is there something different/better about calling it explicitly? I think calling it this way will launch a 2nd make as a sub-task to run it.

File: hack/grade/generate.go [hunk 2]
```diff
@@ -52,17 +52,11 @@ func main() {
...
-
-		//command = exec.Command("mkdir", dashboardPath)
-		//_, err = pkg.RunCommand(command)
-		//if err != nil {
-		//	log.Errorf("running command :%s", err)
-		//}
+		command := exec.Command("mkdir", dashboardPath)
+		_, err = pkg.RunCommand(command)
+		if err != nil {
+			log.Warnf("running command :%s", err)
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply hack/grade/generate.go:58]: We call that in chan and should no fail. These silly fixes are for you folks who are able to run the audit as well.

File: Makefile (hunk 1 above)
Comment (camilamacedo86) [reply Makefile:90]: because I add the docker login there to make it easier for others to understand and to force that is a pre-requirement.

Comment (bentito) [reply Makefile:90]: Those seem like reasonable justifications to doing it this way. 👍

File: hack/grade/generate.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (bentito) [reply hack/grade/generate.go:58]: ah, okay, got it. sounds reasonable

**Pull request merged**